vllm/engine/metrics.py

```diff
@@ -190,6 +190,63 @@ def __init__(self, labelnames: List[str], max_model_len: int):
             labelnames=labelnames,
             multiprocess_mode="sum",
         )
+        
+        self.gauge_cache_service_tokens_hit_rate = self._gauge_cls(
+            name="vllm:gauge_cache_service_tokens_hit_rate",
+            documentation="External cache token hit rate.",
+            labelnames=labelnames,
+            multiprocess_mode="sum")
```

Comment:
Does it make sense to sum up hit rate across multiple processes?

Reply:
All have been replaced with "all"